Classes/DDFileLogger.h

```diff
@@ -302,9 +302,7 @@ extern unsigned long long const kDDDefaultLogFilesDiskQuota;
 /**
  *  The standard implementation for a file logger
  */
-@interface DDFileLogger : DDAbstractLogger <DDLogger> {
-	DDLogFileInfo *_currentLogFileInfo;
-}
+@interface DDFileLogger : DDAbstractLogger <DDLogger>
```

Comment:
This absolutely has to be protected by the logger queue, and hence we can't expose it here in the header. It's been public since 2016, so this is a breaking change.